Resources/Prototypes/Entities/Mobs/NPCs/pets.yml

```diff
@@ -225,6 +225,28 @@
     attributes:
       gender: epicene
 
+- type: entity
+  name: Runtime
+  parent: MobCat
+  id: MobCatRuntime
+  description: Professional mouse hunter. Escape artist.
+  components:
```

Comment:
Everything from this line and below can be removed because they are inherited from the parent. For all the pets below